src/main/java/org/opensearch/ad/AnomalyDetectorJobRunner.java

```diff
@@ -56,23 +73,6 @@
 import org.opensearch.common.xcontent.XContentType;
 import org.opensearch.threadpool.ThreadPool;
 
-import com.amazon.opendistroforelasticsearch.ad.common.exception.AnomalyDetectionException;
-import com.amazon.opendistroforelasticsearch.ad.common.exception.EndRunException;
-import com.amazon.opendistroforelasticsearch.ad.common.exception.InternalFailure;
-import com.amazon.opendistroforelasticsearch.ad.indices.ADIndex;
-import com.amazon.opendistroforelasticsearch.ad.indices.AnomalyDetectionIndices;
-import com.amazon.opendistroforelasticsearch.ad.model.AnomalyDetectorJob;
-import com.amazon.opendistroforelasticsearch.ad.model.AnomalyResult;
-import com.amazon.opendistroforelasticsearch.ad.model.FeatureData;
-import com.amazon.opendistroforelasticsearch.ad.model.IntervalTimeConfiguration;
-import com.amazon.opendistroforelasticsearch.ad.settings.AnomalyDetectorSettings;
-import com.amazon.opendistroforelasticsearch.ad.transport.AnomalyResultAction;
-import com.amazon.opendistroforelasticsearch.ad.transport.AnomalyResultRequest;
-import com.amazon.opendistroforelasticsearch.ad.transport.AnomalyResultResponse;
-import com.amazon.opendistroforelasticsearch.ad.transport.AnomalyResultTransportAction;
-import com.amazon.opendistroforelasticsearch.ad.transport.handler.AnomalyIndexHandler;
-import com.amazon.opendistroforelasticsearch.ad.transport.handler.DetectionStateHandler;
-import com.amazon.opendistroforelasticsearch.ad.util.ClientUtil;
 import com.amazon.opendistroforelasticsearch.commons.InjectSecurity;
```

Comment:
Should we wait for the AD dependencies (`common-utils`, `job-scheduler`) to make the corresponding namespace changes first? Then we can remove all references to `opendistroforelasticsearch`?

Reply:
Good point! 
But considering the tight launch schedule, I prefer to change as much as we could before the AD dependencies were ready. Once the dependencies were ready, we could make another small change to remove all references to opendistroforelasticsearch.

Reply:
line 76 to 83 of this file still using `com.amazon.opendistroforelasticsearch.*`  as `common-utils` and `job-scheduler` not changed yet. 
I think it's ok to change it later when `common-utils` and `job-scheduler` ready. So we can make sure AD build can pass.

Reply:
Cool, it can be fixed after. Created #45 to track.

Reply:
https://github.com/opensearch-project/common-utils/issues/19 and https://github.com/opensearch-project/job-scheduler/issues/22 are being taken of